src/components/more-from-xl-blog/more-from-xl-blog.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
+import { Link } from "gatsby";
+import PropTypes from "prop-types";
+
+import { classnames } from "../../helpers";
+
+import { Card } from "../../components/card";
+
+import * as styles from './more-from-xl-blog.module.scss';
+
+const MoreFromXlBlog = ({ data, refMoreFrom }) => {
+  const { edges } = data.allMdx;
+  return (
+    <div className={styles.moreFromXlContainer} ref={refMoreFrom} >
+      <div className={styles.titleContainer}>
```

Comment:
I think this div is unnecessary

Reply:
I use that div to make the margin top and bottom, because the Link component doesn't apply this style:
![Captura desde 2023-02-06 13-11-46](https://user-images.githubusercontent.com/107500715/217024446-07c8e7e1-9ac3-4923-9854-41d206b4d01b.png)

Reply:
Do not you apply styles in Link in the `titleStyle` class?

Reply:
Yes, I applied styles there, but, the padding top and bottom didn't work. But, if I have a div, they work